### ISSUE TITLE
After installing 2.16.0.1843 nightly, all AdGuard versions crash.

### ISSUE THREAD

**Issue by Miracle-Sakuno** [AdguardTeam/AdguardForMac#1504]
### Please answer the following questions for yourself before submitting an issue

- [X] Filters were updated before reproducing an issue
- [X] I checked the [knowledge base](https://adguard.com/kb/) and found no answer
- [X] I checked to make sure that this issue has not already been filed

### AdGuard version

2.16.0.1843 nightly

### OS version

macOS Sequoia 15.2 Beta (24C5057p)

### Which DNS server do you use?

System default DNS

### DNS protocol

None

### Issue Details

After installing 2.16.0.1843 nightly, all AdGuard versions crash.
I try to install an older version of AdGuard and it crashes too.

### Actual Behavior

[log.txt](https://github.com/user-attachments/files/17591304/log.txt)

**Comment by Aydinv13** [comment]
@Miracle-Sakuno Hi,

could you try the following:
1. Reset the settings via AdGuard icon -> Gear -> Advanced -> Reset Settings
2. Quit AdGuard
3. Remove AdGuard from the Applications directory (move to Bin)
4. Restart your Mac
5. Check that there is no AdGuard in Login Items -> Network Extensions. If so, just [Delete](https://uploads.adguard.org/next_screen.png) the extension
6. Go to SystemExtensions (`cd /Library/SystemExtensions` then type in `ls`) and check that `com.adguard.mac.adguard.network-extension.systemextension` is missing 
7. Install the latest [release](https://agrd.io/mac)

**Comment by Aydinv13** [comment]
@Miracle-Sakuno any news on that?

**Comment by Miracle-Sakuno** [comment]
The problem was solved. After checking, I found that I had executed a command to improve file security some time ago:

```
sudo launchctl config user umask 077
```

The command execution did not cause any problems, but the update or reinstallation caused AdGuard to be unable to write to certain directories, and AdGuard crashed.

```
sudo launchctl config user umask 022
```

I restored the macOS umask defaults after executing this command. Then I completely uninstalled and reinstalled AdGuard. AdGuard worked fine.
